storage/spider/mysql-test/spider/bugfix/r/mdev_24523.result

```diff
@@ -0,0 +1,64 @@
+for master_1
+for child2
+child2_1
+child2_2
+child2_3
+for child3
+
+this test is for MDEV-24523
+
+drop and create databases
+connection master_1;
+CREATE DATABASE auto_test_local;
+USE auto_test_local;
+connection child2_1;
+SET @old_log_output = @@global.log_output;
```

Comment:
@nayuta-yanagisawa Do I need to change the base branch to 10.3? If needed, I will also add the `disable-ps-protocol` and remove `log_output` like my previous fixes.

Reply:
JSON_XXX function is introduce by https://github.com/MariaDB/server/commit/ebe5ebba165863a134c145b1a3709b70925d5100. The commit is also merged into 10.2? So, the bug might reproduce on 10.2. Could you check if it is reproducible on 10.2? The base branch should be the oldest *supported* major version on which the bug reproduces.

Reply:
If the bug is reproducible on 10.2, please set the base branch to 10.2. Adding `--disable-ps-protocol` is also appreciated. Thanks.

Reply:
No problem, I'm taking a day off today, I will work on it tomorrow.